### PR TITLE
Output only machine-parseable info on stdout

### PR DESCRIPTION
**Pull request by kephas**
Output human messages on stderr and ADDR:PORT on stdout for easier
machine parsing

### CODE REVIEW

File: db/tunnel.go [hunk 1]
```diff
@@ -86,7 +86,8 @@ func Tunnel(opts TunnelOpts) error {
 	}
 
 	defer sock.Close()
-	fmt.Printf("You can access your database on '%v'\n", sock.Addr())
+	fmt.Println("You can access your database on:")
```
Comment (Soulou) [inline db/tunnel.go]: Don't you want to display this on stderr also?

**Comment by Soulou** [comment]
@kephas Thanks for the contribution, a small change to match the goal of the PR according to me

File: db/tunnel.go (hunk 1 above)
Comment (kephas) [reply db/tunnel.go]: Ugh, I used grep to test what goes in stdout but I did `grep ":"` and that line ends with one. <facepalm>

Comment (kephas) [reply db/tunnel.go]: *facepalm*

**Pull request merged**